src/front-door/azext_front_door/vendored_sdks/models/__init__.py

```diff
@@ -165,7 +166,7 @@
     'LoadBalancingSettingsModelPaged',
     'BackendPoolPaged',
     'FrontendEndpointPaged',
-    'WebApplicationFirewallPolicyPaged',
+    'WebApplicationFirewallPolicy1Paged',
```

Comment:
This will likely require updates in the code.

Reply:
'WebApplicationFirewallPolicy1Paged' is used internally within the generated SDK. So, no change required in code for this.